worker/worker.go

```diff
@@ -1230,9 +1249,22 @@ func (w *worker) Handler() http.Handler {
 		"GET    /objects/*path": w.objectsHandlerGET,
 		"PUT    /objects/*path": w.objectsHandlerPUT,
 		"DELETE /objects/*path": w.objectsHandlerDELETE,
+
+		"GET    /state": w.stateHandlerGET,
 	}))
 }
 
+func (w *worker) Run() error {
```

Comment:
Can we mimic the `bus` and not have a `Run`? `Bus` and `Worker` don't really "run" like the autopilot so the same pattern doesn't really fit. 
Setting the time in `New` gives the same result and will allow us to not lock the `startTime` because it doesn't come with the same implications as a `Run`.